module-contrib/plugsched.service

```diff
@@ -7,7 +7,9 @@ ConditionKernelCommandLine=!plugsched.enable=0
 
 [Service]
 Type=oneshot
+RemainAfterExit=yes
 ExecStart=/usr/bin/bash -c "/var/plugsched/$(uname -r)/scheduler-installer install"
+ExecStop=/usr/bin/bash -c "/var/plugsched/$(uname -r)/scheduler-installer uninstall"
```

Comment:
1. the commit log is poorly written
2. why need to unload module on service stop?
   https://github.com/dynup/kpatch/commit/8909e63c54adb34a0324200f99c63fcd7db5cbc5

Reply:
I have updated the commit log.

For Kpatch, some hotfix modules cannot be unload because of security reasons. Different from kpatch, for the plugsched, when the scheduler module is installed and uninstalled is optional to users.